modules/openapi-generator/src/main/java/org/openapitools/codegen/utils/ModelUtils.java

```diff
@@ -741,6 +741,16 @@ public static boolean isURISchema(Schema schema) {
                 && URI_FORMAT.equals(schema.getFormat());
     }
 
+    private static boolean isEnumSchema(final Schema<?> schema) {
```

Comment:
can you please refactor https://github.com/OpenAPITools/openapi-generator/blob/e8eee2fd84081935bedd1fcabf73fc53fb79d111/modules/openapi-generator/src/main/java/org/openapitools/codegen/utils/ModelUtils.java#L1384 to use this new function?

Reply:
Done